src/plugin/ipc/ipc.cpp

```diff
@@ -215,10 +215,10 @@ EXTERNC void
 dmtcp_initialize_plugin()
 {
   dmtcp_register_plugin(sshPlugin);
+  dmtcp_register_plugin(eventPlugin);
```

Comment:
I think we can skip this reordering. This is not relevant. The Epoll and File plugins are independent of each other.